### PR TITLE
feat: replace gitcoin chat link with discord

### PR DESCRIPTION
**Pull request by chibie**
##### Description

Replace gitcoin chat links with discord. This is part of revamp toward the discontinuation of gitcoin chat

##### Refers

#7720 #7709

### CODE REVIEW

File: app/dashboard/notifications.py [hunk 1]
```diff
@@ -717,7 +717,7 @@ def maybe_market_tip_to_github(tip):
         addon_msg = f"\n\n * ${amount_usdt_open_work()} in Funded OSS Work Available at: " \
                     f"https://gitcoin.co/explorer\n * Incentivize contributions to your repo: " \
                     f"<a href='https://gitcoin.co/tip'>Send a Tip</a> or <a href='https://gitcoin.co/funding/new'>" \
-                    f"Fund a PR</a>\n * No Email? Get help on the <a href='https://chat.gitcoin.co/'>Gitcoin Chat</a>"
+                    f"Fund a PR</a>\n * No Email? Get help on the <a href='https://discord.com/invite/83BK5z55yB'>Gitcoin Discord</a>"
```
Comment (aamustapha) [inline app/dashboard/notifications.py:720]: I think this should be Gitcoin's Discord. What do you think?

**Review by aamustapha — APPROVED** [review]
Amazing work. Baba show me the way.

File: app/dashboard/notifications.py (hunk 1 above)
Comment (chibie) [reply app/dashboard/notifications.py:720]: oh yea.. that's right!

**Comment by thelostone-mc** [comment]
lol conflicts ! mind fixing them up @chibie and we get this in

**Review by thelostone-mc — APPROVED** [review]

**Pull request merged**